adaptdl/adaptdl/speedup.py

```diff
@@ -72,14 +74,19 @@ def __init__(self, params, grad_params=None, init_batch_size=None,
                                     init_batch_size)
         else:
             self._max_batch_size = init_batch_size
+
+        self._true_max_local_bsz = self._max_local_bsz
+        if gradient_accumulation:
+            self._max_local_bsz = self._max_batch_size
```

Comment:
Confused by this, what's the different between `max_local_bsz` and `true_max_local_bsz` when gradient accumulation is enabled? And why are both required?

Reply:
I need to rename this better tbh. `true_max_local_bsz` is the maximum batch size returned by the dataloader.

Reply:
Removed: changed the logic to a boolean flag